module3/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "module_3_hw",
+  "version": "1.0.0",
+  "description": "module 3 hw",
+  "main": "index.js",
+  "scripts": {
+    "task1": "ts-node --esm src/app.ts"
```

Comment:
To fix all your issues with imports use: "node --experimental-specifier-resolution=node --loader ts-node/esm ./src/app.ts"

Reply:
It really worked. Thanks a lot 🙏